entry/src/entry.rs

```diff
@@ -938,6 +954,22 @@ pub fn next_versioned_entry(
     }
 }
 
+pub fn entry_thread_pool_for_tests() -> ThreadPool {
```

Comment:
I'm fine w/ dropping the entry prefix here since this resides inside the entry module. Similar to what you did for the benches thread pool

Reply:
Ahh thanks, I had the same thought but forgot to make the change